### ISSUE TITLE
Higher validation accuracy

### ISSUE THREAD

**Issue by chengyang317** [sibeiyang/sgmn#6]
Hi, 
Thank you for your great work. When I run the bash experiments 'bash experiments/scripts/train_cmrin.sh 2' and 'bash experiments/scripts/train_dga.sh 1', they show higher validation accuracy compared to your paper. They are 'Epoch: [9]      Loss 0.0899     Prec 0.5305' and 'Epoch: [9]      Loss 0.0847     Prec 0.5283', while your paper shows they are 45.43 and 45.37. I just change the batch size to 48 and the lr as 0.75e-4. Have you encountered this? By the way, I want to test my model in the test split, How can I do such test?

Than you!

**Comment by sibeiyang** [comment]
Hi @chengyang317 ,

Are you using the object features provided by this repository? If yes, the validation accuracy for them is not 45.43 and 45.37 (with Faster-RCNN features), respectively. We provide the bottom-up features here, and the accuracy for models with bottom-up features is given in Table 1 with * (https://sibeiyang.github.io/publication/sgmn/sgmn_cvpr.pdf). BTW, I did not optimize hyperparameters for all the models. 

I will upload the test split to the shared google folder. 

Best,
Sibei

**Comment by wzq12345** [comment]
Hello, I also have a question. Do you use the GQA provided features and then obtain 45.43 and 45.37 for the validation accuracy? If so,  How do you preprocess the validation set? 
I try to calculate the IOU between the reference bounding box (provided by the val expressions) and all of the bounding boxes corresponding to the images (provided by GQA), if the IOU higher than 0.5, which means that detect successfully. Otherwise, detect false. I found that the cases of detecting false are too much, which occupy about 1/3 of the validation referring expressions. 

Thank you!

**Comment by ZhangWenjing1** [comment]
> Hi,
> Thank you for your great work. When I run the bash experiments 'bash experiments/scripts/train_cmrin.sh 2' and 'bash experiments/scripts/train_dga.sh 1', they show higher validation accuracy compared to your paper. They are 'Epoch: [9] Loss 0.0899 Prec 0.5305' and 'Epoch: [9] Loss 0.0847 Prec 0.5283', while your paper shows they are 45.43 and 45.37. I just change the batch size to 48 and the lr as 0.75e-4. Have you encountered this? By the way, I want to test my model in the test split, How can I do such test?
> 
> Than you!

**Comment by ZhangWenjing1** [comment]
> Hi,
> Thank you for your great work. When I run the bash experiments 'bash experiments/scripts/train_cmrin.sh 2' and 'bash experiments/scripts/train_dga.sh 1', they show higher validation accuracy compared to your paper. They are 'Epoch: [9] Loss 0.0899 Prec 0.5305' and 'Epoch: [9] Loss 0.0847 Prec 0.5283', while your paper shows they are 45.43 and 45.37. I just change the batch size to 48 and the lr as 0.75e-4. Have you encountered this? By the way, I want to test my model in the test split, How can I do such test?
> 
> Than you!

**Comment by ZhangWenjing1** [comment]
Hello, when downloading the dataset, the uploader did not give permission. Can you share the dataset you downloaded! thank you

**Comment by ZhangWenjing1** [comment]
Hello, when downloading the dataset, the uploader did not give permission. Can you share the dataset you downloaded!
 Thank you！

**Comment by ZhangWenjing1** [comment]
> Hi @chengyang317 ,
> 
> Are you using the object features provided by this repository? If yes, the validation accuracy for them is not 45.43 and 45.37 (with Faster-RCNN features), respectively. We provide the bottom-up features here, and the accuracy for models with bottom-up features is given in Table 1 with * (https://sibeiyang.github.io/publication/sgmn/sgmn_cvpr.pdf). BTW, I did not optimize hyperparameters for all the models.
> 
> I will upload the test split to the shared google folder.
> 
> Best,
> Sibei

**Comment by ZhangWenjing1** [comment]
Hi,
Thank you for your great work. When I run the bash experiments 'bash experiments/scripts/train_cmrin.sh ' and 'bash experiments/scripts/train_dga.sh ', I want to test my model in the test split, do you upload the test split to the shared google folder.
Than you!

**Comment by ZhangWenjing1** [comment]
> Hi,
> Thank you for your great work. When I run the bash experiments 'bash experiments/scripts/train_cmrin.sh 2' and 'bash experiments/scripts/train_dga.sh 1', they show higher validation accuracy compared to your paper. They are 'Epoch: [9] Loss 0.0899 Prec 0.5305' and 'Epoch: [9] Loss 0.0847 Prec 0.5283', while your paper shows they are 45.43 and 45.37. I just change the batch size to 48 and the lr as 0.75e-4. Have you encountered this? By the way, I want to test my model in the test split, How can I do such test?
> 
> Than you!
